src/client.h

```diff
@@ -73,6 +73,9 @@
 // audio reverberation range
 #define AUD_REVERB_MAX 100
 
+// delay period between successive gain updates (ms)
+#define GAIN_DELAY_PERIOD_MS 300
```

Comment:
No, someone moving faders would be hard pushed to notice a 300ms delay in the sound changing, but it could take nearly that long for the message to be acked from a very distant server. The aim is not to generate and queue messages faster than they could be dequeued for sending. Message N+1 is not sent until message N has been acked. So during the delay period, we just keep track of updates to the requested value and then send only the latest one when the timer fires.

Reply:
I guess if it does become noticed, it's easy enough to adjust.

Reply:
And if a single update is requested after an idle period (e.g. clicking Mute, or clicking the Fader straight to a target position, if the GUI allows it), then it will still get sent immediately, and after the time fires it will just notice there has been no change.